exonum/src/storage/key_set_index.rs

```diff
@@ -154,7 +154,7 @@ where
     /// ```
     pub fn iter(&self) -> KeySetIndexIter<K> {
         KeySetIndexIter {
-            base_iter: self.base.iter(&()),
+            base_iter: self.base.iter(&(), None as Option<&[u8]>),
```

Comment:
As for me, syntax became less readable.

Reply:
Is `as Option<&[u8]>` part really needed here?..

Reply:
Unfortunately, yes